composer.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "heyloyalty/hl-phpclient",
+  "name": "rochadk/hl-phpclient",
```

Comment:
It shouldn't. I only changed it to be able to pull my repo with composer - I've made it backwards compatible, because our setup is ancient. Actually, only the first change should be merged into your repo.